status_dashboard.py

```diff
@@ -0,0 +1,33 @@
+from retriever import install_sqlite
+from retriever.lib.engine_tools import getmd5
+from retriever import datasets
+from retriever.engines import engine_list
+from tempfile import mkdtemp
+from shutil import rmtree
+import os
+
+mysql_engine, postgres_engine, sqlite_engine, msaccess_engine, csv_engine, download_engine, json_engine, xml_engine = engine_list
+
+file_location = os.path.dirname(os.path.realpath(__file__))
+
+example_datasets = ['bird-size', 'mammal-masses', 'airports', 'elton-traits']
+precalculated_md5 = {'airports': 'f16a0bb21d347e2ba180cde26f35c87a',
+                     'bird-size': '98dcfdca19d729c90ee1c6db5221b775',
+                     'mammal-masses': '6fec0fc63007a4040d9bbc5cfcd9953e',
+                     'elton-traits': '9da42cd158074d68eb464e8c7b37d3ee'}
+
+for script in datasets():
+    if script.name in example_datasets:
+        workdir = mkdtemp(dir=file_location)
+        os.chdir(workdir)
+        install_sqlite(script.name, use_cache=False, file=os.path.join(file_location, 'new_sqlite.db'))
+        engine_obj = script.checkengine(sqlite_engine)
+        engine_obj.to_csv()
+        current_md5 = getmd5(os.getcwd(), data_type='dir')
+        os.chdir(file_location)
+        os.remove("new_sqlite.db")
+        rmtree(workdir)
+        if current_md5 == precalculated_md5[script.name]:
+            print('Passed')
+        else:
+            print("Current md5 of {} is {} expected {}".format(script.name, current_md5, precalculated_md5[script.name]))
```

Comment:
If a script's md5 sum is not the same could we add some function that tries to identify why we are having different md5 hashes. 
This could be because the files did not get downloaded or they are missing(url 404 error ) Or the files changed by adding more data or renaming.

Reply:
Refer to the `test_modified script` to see how we install and make sure that there are no errors. If there are error we should be able to identify the errors.